.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest] #, windows-latest]
+        os: [ubuntu-latest, macos-12] #, windows-latest]
```

Comment:
Why is that? Is this reflected in e2e changes in client?

Reply:
Some dependency started to fail due to minimal requirement of MacOS (from master: https://github.com/neptune-ai/neptune-fastai/actions/runs/4111264114/jobs/7278991104#step:4:906). I was trying to fix that by enforcing the latest Github Runner image. Sorry, I forget to remove this change before posting a request for code review :wink: